package.json

```diff
@@ -53,6 +55,13 @@
     "lint:sol:ci": "prettier --check contracts/*.sol contracts/**/*.sol && solhint **/*.sol --max-warnings 0",
     "lint:ts:ci": "prettier --check ./**/*.ts",
     "prepare": "husky install",
+    "keys:decrypt": "bash scripts/key_placer.sh decrypt",
+    "keys:encrypt": "bash scripts/key_placer.sh encrypt",
+    "deploy:alfajores": "hardhat stakedCelo:deploy  --network alfajores --show-stack-traces --tags core",
+    "deploy:staging": "hardhat stakedCelo:deploy --network staging --show-stack-traces --tags core",
+    "deploy:mainnet": "hardhat stakedCelo:deploy  --network mainnet --show-stack-traces --tags core",
```

Comment:
Wondering if we need these separate commands or just leave it up to the user to specify `--network`, since that's the only difference between these.